modules/cnf-numa-resource-scheduling-strategies.adoc

```diff
@@ -0,0 +1,71 @@
+
+. After a worker node is selected, the scheduler evaluates its NUMA nodes and applies a scoring strategy to decide which NUMA node will handle the workload.
+
+. After a workload is scheduled, the selected NUMA node’s resources are updated to reflect the allocation.
+
+The default strategy applied is the `LeastAllocated` strategy. This assigns workloads to the NUMA node with the most available resources that is the least utilized NUMA node. The goal of this strategy is to spread workloads across NUMA nodes to reduce contention and avoid hotspots.
+
+[discrete]
+[id="cnf-leastallocated-example_{context}"]
+== LeastAllocated strategy (default) example
+This strategy assigns workloads to the NUMA node with the most available resources, minimizing resource contention and spreading workloads across NUMA nodes. This reduces hotspots and ensures sufficient headroom for high-priority tasks. Assume a worker node has two NUMA nodes, and the workload requires 4 vCPUs and 8 GB of memory:
+
+[cols="5,2,2,2,2,2", options="header"]
+|===
+|NUMA node |Total CPUs |Used CPUs |Total memory (GB) |Used memory (GB) |Available resources
+|NUMA 1 |16 |12 |64 |56 |4 CPUs, 8 GB memory
+|NUMA 2 |16 |6 |64 |24 |10 CPUs, 40 GB memory
+|===
+
+NUMA 2 has more available resources (10 CPUs, 40 GB memory) compared to NUMA 1 (4 CPUs, 8 GB memory), therefore the workload is assigned to NUMA 2.
+
+[discrete]
+[id="cnf-mostallocated-example_{context}"]
+== MostAllocated Strategy example
+This strategy consolidates workloads by assigning them to the NUMA node with the least available resources, which is the most utilized NUMA node. This approach helps free other NUMA nodes for energy efficiency or critical workloads requiring full isolation. Using the same initial NUMA node states as earlier:
+
+[cols="2,2"]
+|===
+|NUMA node |Available resources
+|NUMA 1 |4 CPUs, 8 GB memory
+|NUMA 2 |10 CPUs, 40 GB memory
+|===
+
+The workload again requires 4 vCPUs and 8 GB memory. NUMA 1 has fewer available resources compared to NUMA 2, so the scheduler consolidates the workload on NUMA 1. The workload is assigned to NUMA 1, further utilizing its resources while leaving NUMA 2 idle or minimally loaded.
+
+[discrete]
+[id="cnf-balanceallocated-example_{context}"]
+== BalancedAllocation strategy example
+This strategy assigns workloads to the NUMA node with the most balanced resource utilization across CPU and memory. The goal is to prevent imbalanced usage, such as high CPU utilization with underutilized memory. Assume a worker node has the following NUMA node states:
+
+[cols="2,2,2,2"]
+|===
+|NUMA node |CPU usage |Memory usage |`BalancedAllocation` score
+|NUMA 1 |60% |55% |High (more balanced)
+|NUMA 2 |80% |20% |Low (less balanced)
+|===
+
+NUMA 1 has a more balanced CPU and memory utilization compared to NUMA 2 and therefore, with the `BalancedAllocation` strategy in place, the workload is assigned to NUMA 1.  
+
```

Comment:
So table needs proper title; beyond that, I would give this summary **first** and make it a section with a heading.  Perhaps right above the LeastAllocated section. Much more scannable for advanced users who just need to reference the principles. You can provide the detailed breakdown after.